aqua/input-leap/Portfile

```
@@ -0,0 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        input-leap input-leap 198779cf9e798aac6ad41427e32d031305736de3
+version             20230416
```

Perhaps we should keep the real version number in here so that when we update to the next stable version we don't have to increase `epoch`, and so that the user knows what version this port's code is based on.
```suggestion
version             2.4.0-20230416
```

---

> I am assuming here that there is a good reason why we have to use some arbitrary commit rather than the latest stable version like we usually do.

Correct. Input-leap was forked from Barrier ~2 years ago since latter was (and still is) unmaintained. But for whatever reason, input-leap maintainers have not done a single official release since forking. I'm trying to help improve project visibility by creating this package.